plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/shipping/test/index.tsx

```diff
@@ -368,57 +368,9 @@ describe( 'TotalsShipping', () => {
 			screen.queryByText( 'Change address' )
 		).not.toBeInTheDocument();
 		expect(
-			screen.getByText( 'Add an address for shipping options' )
+			screen.getByText( 'Enter address to check delivery options' )
 		).toBeInTheDocument();
 	} );
-	it( 'does not show the calculator button when default rates are available and no address has been entered', () => {
```

Comment:
This test checked for presence of a string that wasn't in the codebase. It also doesn't seem like a valid test. Why would we want to remove the button just because default rates are available?